### PR TITLE
AudioManager can now play sounds + stop all sounds

### PR DESCRIPTION
**Pull request by gterminator**
AudioManager can now play sounds + it can stop all sounds if you want to.

Todos:
- Readme
- There is a check that only the same sound can be played at a time. ( Maybe there is room for improvement)

### CODE REVIEW

**Comment by bitbrain** [comment]
Good stuff!

File: core/src/main/java/de/bitbrain/braingdx/audio/AudioManager.java [hunk 1]
```diff
@@ -124,6 +131,44 @@ public void stopMusic(String path) {
...
+         throw new RuntimeException("Sound: key: " + key + " path: " + path + " already playing");
+      }
+
+      Sound sound = Gdx.audio.newSound(Gdx.files.internal(path));
+      soundMap.put(key, sound);
+      if (loop) {
+         sound.loop();
+      }
+      else {
+         sound.play();
```
Comment (bitbrain) [inline core/src/main/java/de/bitbrain/braingdx/audio/AudioManager.java]: Could we do `sound.play(volume);` instead?

Comment (gterminator) [inline core/src/main/java/de/bitbrain/braingdx/audio/AudioManager.java]: yes good idea

**Comment by gterminator** [comment]
Better close this pull request. This implementation has a bug you mentioned. There is no way in gdx to find out that the sound is complete. Better you music "All the time".

**Pull request closed without merging**